keymanapp/keyman
# chore(common): Add crowdin strings for Dutch

This adds the crowdin strings for Nederlands (Dutch) - locale nl-NL

TODO for @sgschantz 

## User Testing
Load the PR build for each platform and set the UI / Locale to Dutch
There may be a handful of strings still in English, but most are in Dutch

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> Nederlands (Dutch)
3. Return to the Keyman app 
4. Verify the UI is in Dutch (prompt, Settings strings, etc.). 

* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Dutch
3. Return to the Keyman app
4. Verify the UI is in  Dutch

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=nl_NL km-config
2. Verify Keyman UI is in Dutch

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the locale to Dutch
3. Verify the Keyman UI is in Dutch

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to Nederlands
3. Verify the Keyman UI is in Dutch
